### DISCUSSION TITLE
How do I use the Proxmox VE Post Install script on PVE 8.3.0?

### DISCUSSION THREAD

**Discussion by small-grey-dog** [community-scripts/ProxmoxVE#498] [Q&A]
How do I get the Proxmox VE Post Install script working on PVE 8.3.0? I try to run the command in the shell and get the following error:

> Start the Proxmox VE Post Install Script (y/n)?y
>  ✗ This version of Proxmox Virtual Environment is not supported
> Requires Proxmox Virtual Environment Version 8.0 or later.
> Exiting...
> root@xyz:~#

Thanks in advance :)

**Comment by MickLesk** [comment]
working since a few days. Reset Your Console and check next time the issues and discussions

**Reply by small-grey-dog** [reply]
Sorry, I don't use Github much. I'm new here haha.

**Reply by small-grey-dog** [reply]
FYI the website info is incorrect: https://tteck.github.io/Proxmox/#proxmox-ve-post-install

**Reply by MickLesk** [reply]
what do you mean? The missing info? Thats the old website, not updated.
use helper-scripts.com or https://community-scripts.github.io/ProxmoxVE/
